src/test/java/org/prebid/server/bidder/colossus/ColossusBidderTest.java

```diff
@@ -103,7 +106,7 @@ public void makeHttpRequestsShouldMakeOneRequestPerImp() {
                 .extracting(httpRequest -> mapper.readValue(httpRequest.getBody(), BidRequest.class))
                 .flatExtracting(BidRequest::getImp).hasSize(2)
                 .extracting(Imp::getTagid)
-                .containsOnly("tagidString", "otherTagId");
+                .containsOnly("tagidStr", "otherTagId");
```

Comment:
Can you please change `.containsOnly` into `.containsExactly`? In whole bidder tests file where possible.